gpt_computer_agent/__init__.py

```
@@ -2,6 +1,288 @@
+
+    def request(self, the_request, the_response, screen=False):
+        screen = "false" if not screen else "true"
+
+        response = requests.post(self.url+"request", data={"request": the_request, "response": the_response, "screen":screen, "instance":self.instance_id}, verify=True)
+        json_response = response.json()
+        request_id = json_response["request_id"]
+        try:
+            while True:
+                response = requests.post(self.url+"request_result", data={"request_id": request_id}, verify=True)
+                the_json = response.json()
+                if the_json["status"] == True:
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if the_json["status"]:` for truth checks

Replace with `the_json["status"]`

(E712)